app/code/Magento/Vault/Model/PaymentTokenFactory.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Vault\Model;
+
+use Magento\Framework\ObjectManagerInterface;
+use Magento\Vault\Api\Data\PaymentTokenFactoryInterface;
+use Magento\Vault\Api\Data\PaymentTokenInterface;
+
+/**
+ * PaymentTokenFactory class
+ * @api
+ */
+class PaymentTokenFactory implements PaymentTokenFactoryInterface
+{
+    /**
+     * @var array
+     */
+    private $tokenTypes = [];
+
+    /**
+     * PaymentTokenFactory constructor.
+     * @param ObjectManagerInterface $objectManager
+     * @param array $tokenTypes
+     */
+    public function __construct(ObjectManagerInterface $objectManager, $tokenTypes = [])
```

Comment:
I don't know what you mean here. array has been definied in the doc comments and is specified in the param as default value was well. What do you want to be described?